lib/RoleDetails/RoleDetails.js

```diff
@@ -51,6 +51,7 @@ import css from './style.css';
 
 export const RoleDetails = ({
   displayShareAction = false,
+  isClickableRolesUsers = false,
```

Comment:
To be honest, the prop's name is a bit confusing to me. Just looking at the name, I can't figure out what it is used for. I'd suggest `isUserLinkEnabled` or smth like that.

Reply:
Or `isUserNameClickable`